nominal/core/client.py

```diff
@@ -963,6 +965,36 @@ def get_data_review(self, rid: str) -> DataReview:
         response = self._clients.datareview.get(self._clients.auth_header, rid)
         return DataReview._from_conjure(self._clients, response)
 
+    def create_event(
+        self,
+        name: str,
+        asset_rids: Sequence[str],
+        start: datetime | IntegralNanosecondsUTC,
```

Comment:
can we use the same list of timestamp types we use everywhere else?

Reply:
can you be more specific? the library seems to alternately use `datetime | IntegralNanosecondsUTC` (as above) or `str | datetime | IntegralNanosecondsUTC`

Reply:
ah you're right, I think I initially aimed for str support only in the top-level functions and not the core, but it's probably diffused ovver time